src/schema/imported/theme.json

```diff
@@ -194,7 +194,15 @@
               "maxItems": 15
             },
             "headerURL": {
-              "$ref": "manifest#/types/ImageDataOrExtensionURL"
+              "allOf": [
+                {
+                  "$ref": "manifest#/types/ImageDataOrExtensionURL"
+                },
+                {
+                  "$comment": "This part has been imported manually and it is expected to be overwritten once the changes to the Firefox schemas reach beta",
```

Comment:
I imported this manually from a local mozilla-central tree, so that this PR is complete and it can be reviewed, and also because this way we can start to the "deprecated LWT aliases" warning messages even before the change on the mozilla-central side are going to reach beta (and I added some jsonschema comments so that we are going to be aware that these are meant to be overwritten on the next schema updates).

Reply:
Potentially file a "Import Firefox 65 schema" issue and make a note about that there so that whoever does it, reads it, I'm not sure I'd look for `$comment` in the schema tbh.

Reply:
I already rebased this PR on top of the Firefox 65 imported schema, and so this has been resolved.